ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -342,6 +356,240 @@ objects:
+
+          echo "Create two masters via Machine API"
+          retry 10 oc get machines -n openshift-machine-api
+          # Clone existing masters, update IDs and oc apply
+          SURVIVING_MASTER_MACHINE=$(oc get machine -l machine.openshift.io/cluster-api-machine-role=master -n openshift-machine-api -o name | cut -d '/' -f 2)
+          SURVIVING_MASTER_NUM=${SURVIVING_MASTER_MACHINE##*-}
+          SURVIVING_MASTER_PREFIX=${SURVIVING_MASTER_MACHINE%-*}
+          oc get --export machine ${SURVIVING_MASTER_MACHINE} -n openshift-machine-api -o yaml > /tmp/machine.template
+          SURVIVING_MASTER_AZ=$(grep -oP 'availabilityZone: (\K.+)' /tmp/machine.template)
+          SURVIVING_MASTER_AZ_INDEX=$(grep -oP 'availabilityZone: .*(\K.)' /tmp/machine.template)
+
+          MASTER_INDEX=0
+          AZ_LIST=("a" "b" "c")
+          for i in "${!AZ_LIST[@]}"; do
+            if [[ "${AZ_LIST[$i]}" == "${SURVIVING_MASTER_AZ_INDEX}" ]]; then continue; fi
+            if [[ "${MASTER_INDEX}" == "${SURVIVING_MASTER_NUM}" ]]; then MASTER_INDEX=$((MASTER_INDEX+1)); fi
+            cat /tmp/machine.template \
+              | sed 's;selfLink.*;;g' \
+              | sed "s;name: ${SURVIVING_MASTER_PREFIX}-${SURVIVING_MASTER_NUM};name: ${SURVIVING_MASTER_PREFIX}-${MASTER_INDEX};" > /tmp/machine_${i}.yaml
+              for j in $(seq 0 10); do oc create -n openshift-machine-api -f /tmp/machine_${i}.yaml && break; done
+              MASTER_INDEX=$((MASTER_INDEX+1))
+          done
+
+          echo "Waiting for machines to be created"
+          NEW_MASTER_IPS=()
+          for i in $(seq 0 60); do
+            NEW_MASTER_IPS=($(oc -n openshift-machine-api \
+              get machines \
+              -l machine.openshift.io/cluster-api-machine-role=master \
+              -o jsonpath='{range .items[*]}{.status.addresses[?(@.type=="InternalIP")].address}{"\n"}{end}' || true))
+            if [[ "${#NEW_MASTER_IPS[@]}" == "3" ]]; then break; fi
+            sleep 30
+          done
+          oc get machines -n openshift-machine-api
+          if [[ "${#NEW_MASTER_IPS[@]}" != "3" ]]; then
+            echo "${NEW_MASTER_IPS[@]}"
+            exit 1
+          fi
+
+          echo "Verify new master nodes have joined the cluster"
+          FOUND_MASTERS=0
+          for i in $(seq 1 60)
+          do
+            FOUND_MASTERS=($(oc --request-timeout=5s get nodes -l node-role.kubernetes.io/master= -o name --no-headers || true))
+            if [[ "${#FOUND_MASTERS[@]}" == "3" ]]; then break; fi
+            sleep 30
+          done
+          oc get nodes
+          if [[ "${#FOUND_MASTERS[@]}" != "3" ]]; then
+            echo "${FOUND_MASTERS[@]}"
+            exit 1
+          fi
+
+          echo "Update DNS and LB"
+          # aws cli magic
+          easy_install --user 'setuptools==12.0'
+          easy_install --user boto3
+          cat > /tmp/update_route_53.py <<-PYTHON_EOF
```

Comment:
You're not actually using the tab-indents that `<<-` allows, are you?  Might as well stick with `<<PYTHON_EOF`.

Reply:
Right, fixed in a60bc18b4